app/src/main/java/com/android/wardrobeManager/ui/images/filters/HorizontalStripes.java

```diff
@@ -11,6 +14,7 @@
 
     protected Map<String, DesignFilterManager.DesignFilter> filters;
     private int thickness;
+    private int colorIndex;
```

Comment:
Does this have to be a member variable? Can't you just define it locally in the filter function?